_benchmark/user-guide/target-throughput.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Target throughput
+nav_order: 150
+---
+
+# Target throughput
+
+Target throughput is key to understanding the OpenSearch Benchmark definition of **latency**. Target throughput is the rate at which OpenSearch Benchmark issues requests, assuming that responses will be returned instantaneously. `target-throughput` is one of the common workload parameters that can be set for each test and is measured in operations per second.
+
+There are two types of testing modes when using OpenSearch Benchmark, both of which are related to throughput, latency, and service time:
+
+- [Benchmarking mode](#benchmarking-mode): Latency is measured the same as service time.
+- [Throughput-throttled mode](#throughput-throttled-mode): Latency is service-time plus the time a request spends waiting in the queue.
+
+## Benchmarking mode
+
+When you do not specify a `target-throughput`, OpenSearch Benchmark latency tests are performed in **Benchmarking mode**. In **Benchmarking mode**, the OpenSearch client sends requests to the OpenSearch cluster as fast as possible. After the cluster it receives a response from the prior request sent, Benchmark sends the next request immediately to the OpenSearch client without waiting. In this testing mode, latency is identical to service time.
+
+## Throughput-throttled mode
+
+**Throughput** measures the rate at which OpenSearch Benchmark issues requests, assuming that responses will be returned instantaneously. However, users can set a `target-throughput` is one of the common workload parameters that can be set for each test and is measured in operations per second.
+
+OpenSearch Benchmark always issues one request at a time for a single client thread, specified as `search-clients` in the workload parameters. If `target-throughput` is set to `0`, OpenSearch Benchmark issues a request immediately after it receives the response from the previous request. If the `target-throughput` is not set to `0`, OpenSearch Benchmark issues the next request to match the `target-throughput`, assuming that responses are returned instantaneously.
+
+When you want to simulate the type traffic you might encounter when deploying a production cluster, set the `target-throughput` in your benchmark test to match to the rate of requests you think the production cluster might receive. The following examples illustrate how the set `target-throughput` affects the latency measurement.
```

Comment:
```suggestion
When you want to simulate the type traffic you might encounter when deploying a production cluster, set the `target-throughput` in your benchmark test to match to the request rate you think the production cluster might receive. The following examples show how the set `target-throughput` affects the latency measurement.
```

Reply:
There is no such thing as "request rate", which implies a measurement similar to that of "service time" or "processing time." I'll switch to "number of requests".